requirements.txt

```diff
@@ -1,16 +1,10 @@
-# WARNING
```

Comment:
Gonna assume you know what you’re doing removing this.

Reply:
yeah, dockerfile was updated to automatically read the requirements.txt and install it